smart-connector-rest-server/src/main/java/eu/knowledge/engine/rest/api/impl/ProactiveApiServiceImpl.java

```diff
@@ -127,6 +129,141 @@ public void scAskPost(
 			return;
 		}
 
+		try {
```

Comment:
It is a bit confusing that it currently shows so many (green) changes in this file, while in reality not *that* much has changed. Maybe this is caused by the old `scAskOldPost(...)` still being there?

Reply:
true, I will delete the scAskOldPost